### ISSUE TITLE
Delay after tapping piano key

### ISSUE THREAD

**Issue by RichAJacobs** [AppleEducate/flutter_piano#5]
There is a significant delay after I tap a key on the piano before I hear the sound. 1/4 to 1/2 a second.

My phone is a Pixel 3 XL with Android 9. I tried turning off hepatics in device settings and turning off feedback in the piano app settings. 

I have version 1.0.2 of the Pocket Piano. Thanks.

**Comment by rodydavis** [comment]
Thank you for the feedback!

I did somethings to try to resolve the delay son Android. Does turning off the key feed back help? I tested on the original Google Pixel

**Comment by MrHazee** [comment]
Hi there and thank you for this great project! Really awesome!
I´m also wondering about the latency pissue, any news regarding this matter? It would be so awesome if we could bring the latency down
Maybe audioplayers can be used instead?  
Thanks

**Comment by rodydavis** [comment]
Audio players cannot be used as I’m actually generating the sounds from a midi synth and not simply playing back audio files.

I believe it has something to do with the audio buffer, but no solution yet.

**Comment by MrHazee** [comment]
Yes I meant in that case to hook the piano keys up with audio files instead! Thanks

**Comment by rodydavis** [comment]
The problem with that is that it will not be dynamic, meaning features like switching piano sounds will not work. In addition to bundle 88 .wav/.mp3 files will take up so much more space than midi files.

**Comment by SniffyMcTasty** [comment]
Hi there!

I'm currently learning Flutter/Dart and stumbled upon your code. I've been trying to recreate it myself and I'm having the same issue. Has this been resolved, or is the delay still a problem? I'm guessing it has something to do with how your flutter_midi plugin works with or loads Bitmaps from MIDI files, but I'm not knowledgeable enough to know what part of it does that.
